datalad/support/keyring_.py

```diff
@@ -46,6 +48,7 @@ def _get_service_name(cls, name):
     def get(self, name, field):
         # consult environment, might be provided there and should take precedence
         env_var = ('DATALAD_%s_%s' % (name, field)).replace('-', '_')
+        lgr.log(5, 'Credentials lookup attempt %s', env_var)
```

Comment:
```suggestion
        lgr.log(5, 'Credentials lookup attempt via env var %s', env_var)
```
to make it more explicit?